cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/LimitTest.scala

```diff
@@ -76,6 +78,36 @@ class LimitTest extends DocumentingTest {
         resultTable()
       }
     }
+    section("`LIMIT` will not stop side effects", "limit-will-not-stop-side-effects") {
+      p(
+        """The use of LIMIT in a query will not stop side effects, like `CREATE`, `DELETE` or `SET`, from happening. This behaviour was undefined in versions before 4.3.""".stripMargin
```

Comment:
Maybe add a sentence and example saying that if you want to control updates you need to do something like 

```
MATCH (n {name: 'A'})
WITH n LIMIT 0
SET n.age = 60
RETURN n
```